Impossible to set variable server_capabilities with value "569867". Resetting to current "569899"

My environment uses the latest proxysql version 2.0.4-116-g7d371cf, the back-end proxy 3 Galera cluster (MariaDB 10.3.12 stable), because occasionally appears.

```
Timeout on Galera health check for host after 804ms.
```
 so I added mysql-monitor_galera_healthcheck_timeout to the prompt and enabled the parameter to take effect. At the time of entry into force, the following warnings appear in the Journal:
```
[WARNING] Impossible to set variable server_capabilities with value "569867". Resetting to current "569899".
```
I saw this parameter, but I didn't quite understand the explanation in the document. Was there any problem with the default configuration I used?

Proxysql I use docker container to deploy and use CentOS Linux release 7.2.1511

At the same time, I still have a problem. Even if I increase timeout, proxysql will still make mistakes.
```
2019-06-14 14:11:15 MySQL_Monitor.cpp:1782:monitor_galera_thread(): [ERROR] Got error. mmsd 0x7fb8ede0cd20 , MYSQL 0x7fb8ed401900 , FD 106 : timeout check
2019-06-14 14:11:15 MySQL_Monitor.cpp:1532:monitor_galera_thread(): [ERROR] Timeout on Galera health check for 100.119.146.218:3317 after 2001ms. If the server is overload, increase mysql-monitor_galera_healthcheck_timeout.
2019-06-14 14:11:15 MySQL_Monitor.cpp:1782:monitor_galera_thread(): [ERROR] Got error. mmsd 0x7fb8ede0dd60 , MYSQL 0x7fb8ed804600 , FD 118 : timeout check
```

--- a/lib/MySQL_Thread.cpp
+++ b/lib/MySQL_Thread.cpp
@@ -1879,8 +1879,12 @@ bool MySQL_Threads_Handler::set_variable(char *name, const char *value) {	// thi
 		}
 	}
 	if (!strcasecmp(name,"server_capabilities")) {
-		int intv=atoi(value);
-		if (intv > 10 && intv <= 65535) {
+		// replaced atoi() with strtoul() to have a 32 bit result
+		uint32_t intv = strtoul(value, NULL, 10);
+		if (intv > 10) {
+			// Note that:
+			// - some capabilities are changed at runtime while performing the handshake with the client
+			// - even if we support 32 bits capabilities, many of them do not have any real meaning for proxysql (not supported)
 			variables.server_capabilities=intv;
 			return true;
 		} else {

